test/bin/find.js

```diff
@@ -40,6 +40,12 @@ describe('bin', function() {
     delete require.cache[require.resolve('yargs')]
   })
 
+  it('no option', function() {
```

Comment:
fixed 
- https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127286810#L261
- https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127286811#L193
but it print the `console.logs` too.